pyctcdecode/language_model.py

```diff
@@ -19,6 +21,10 @@
     LOG_BASE_CHANGE_FACTOR,
 )
 
+DEFAULT_MODEL_FILE_NAME = "kenLM.arpa"
+LIBRARY_NAME = "pyctcdecode"
+CACHE_DIRECTORY = os.path.join(Path.home(), ".cache", LIBRARY_NAME)
```

Comment:
Does HF have a recommendation here for where to cache things? Is it more appropriate to have it be `~/.cache/pyctcdecode/hf-hub` ? 
(honestly asking, no strong feelings)

Reply:
Lots of pip repos use `~/.cache/` to cache things. So we do the same in Hugging Face. Then we add the full github path to the `.cache` so that it looks like `~/.cache/huggingface/transformers/...` and `~/.cache/huggingface/datasets/...`. 

We could do the same here and setting the cache to `~/.cache/kensho-technologies/pyctcdecode/...` or we could also do `~/.cache/pyctcdecode/hf-hub` - whatever you prefer :-)